0019/README.md

```diff
@@ -0,0 +1,129 @@
+---
+name: Metadata Transport Protocol
+shortname: 19/MDTP
+status: raw
+editor: Vitalii Kozlovskyi <vkozlovskyi@kaaiot.io>
+---
+
+
+<!-- toc -->
+
+
+# Introduction
+
+Metadata Transport Protocol (MDTP) is designed to request device metadata from Endpoint Registry (EPR) and receive.
```

Comment:
Metadata Transport Protocol (MDTP) is designed to request and receive device metadata from Endpoint Registry (EPR).